tidb-cloud/v7.1.0-performance-benchmarking-with-sysbench.md

```diff
@@ -0,0 +1,175 @@
+---
+title: TiDB Cloud Sysbench Performance Test Report
+summary: Introduce the Sysbench performance test results for TiDB Cloud.
+---
+
+# TiDB Cloud Sysbench Performance Test Report
+
+This document provides the Sysbench performance test steps and results for a TiDB Dedicated cluster with the TiDB version of v7.1.0. This report can also be used as a reference for the performance of TiDB v7.1.0 clusters.
+
+## Test overview
+
+This test aims at showing the Sysbench performance of TiDB v7.1.0 in the Online Transactional Processing (OLTP) scenario.
+
+## Test environment
+
+### TiDB cluster
+
+The test is conducted on a TiDB cluster with the following settings:
+
+- Cluster type: [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated)
+- Cluster version: v7.1.0
+- Cloud provider: AWS (us-west-2)
+- Cluster configuration:
+
+   | Node type | Node size | Node quantity | Node storage |
+   |:----------|:----------|:----------|:----------|
+   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+   | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
+
+### Parameter configuration
+
+> **Note:**
+>
+> To modify the TiKV configuration of your cluster, you can contact the [PingCAP support team](/tidb-cloud/tidb-cloud-support.md) for help.
+
+The TiKV parameter [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) can make log recycling effective immediately after initialization.
+
+Enable the [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) parameter in `oltp_insert`, `oltp_read_write` , `oltp_update_index`, and `oltp_update_non_index` tests:
+
+```yaml
+raft-engine.prefill-for-recycle = true
+```
+
+Disable the [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) parameter in the `oltp_point_select` test:
+
+```yaml
+raft-engine.prefill-for-recycle = false
+```
```

Comment:
````suggestion
The TiKV parameter [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) can make log recycling effective immediately after initialization. This document conducts tests based on different parameter values across several workloads:

- For the `oltp_point_select` workload, disable the [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) parameter:

    ```yaml
    raft-engine.prefill-for-recycle = false
    ```

- For `oltp_insert`, `oltp_read_write` , `oltp_update_index`, and `oltp_update_non_index` workloads, enable the [`prefill-for-recycle`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#prefill-for-recycle-new-in-v700) parameter:

    ```yaml
    raft-engine.prefill-for-recycle = true
    ```
````